man/nvim.1

```diff
@@ -367,11 +381,11 @@ Has no effect in GUIs.
 .El
 .Sh FILES
 .Bl -tag -width Pa
-.It Pa ~/.nvimrc , ~/.nvim/nvimrc
+.It Pa $XDG_CONFIG_HOME/nvim/init.vim
```

Comment:
I'm 80% sure we should stick to `~/.config/nvim/...` and avoid XDG variables everywhere except in one specific section of the user manual (and also perhaps this manpage) which mentions _for those interested_ that we support `$XDG_CONFIG_HOME` and friends.